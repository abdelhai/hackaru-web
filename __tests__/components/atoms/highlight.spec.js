import Factory from '@/__tests__/__setups__/factory';
import Highlight from '@/components/atoms/highlight';

describe('Highlight', () => {
  it('render correctly', () => {
    const wrapper = new Factory(Highlight, {
      slots: {
        default: '<p>Highlight text</p>'
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
