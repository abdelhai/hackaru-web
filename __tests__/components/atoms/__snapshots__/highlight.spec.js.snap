// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Highlight render correctly 1`] = `
<div
  class="highlight"
>
  <p>
    Highlight text
  </p>
</div>
`;
